README.md

```diff
@@ -3,35 +3,82 @@
 ## Overview
 The edX learning platform's frontend for enterprise learners.
 
-## Getting started
+## Setup for development
 
-- Clone this repository locally with `git clone <repo URL>`
+Install nodejs (using nvm maybe a good idea to be able to switch node versions and because there is an .nvmrc file that will be usable by nvm)
 
-##### How to set up the dev environment
 To run the project, first install dependencies
 
-`$ npm i`
+`$ nvm use` # will install/setup node needed for this project
+`$ npm i`   # will install deps
 
 then, start the dev server:
 
 `$ npm start`
 
-### Where is it running?
-The project itself will run on http://localhost:8734
+Visit http://localhost:8734
 
-### Setup prerequisites for full development
+At this point, it will ask you to login. Use the user indicated in devstack (edx@example.com, edx). When logged in, expect to see a 404 since we are not setup
 
-To setup test data, and other backend, please follow these steps, you will setup a `test-enterprise` customer who will have learners associated with it:
+Now quickly check the .env.development file for which services are used by the application. e.g. `LMS_BASE_URL='http://localhost:18000'` . We will get back to this.
 
- - Start backend services needed (refer to .env.development file). Refer to [Devstack Doc](https://github.com/edx/devstack) for details
- - Seed test data. Refer to [The Enterprise management commands](https://github.com/edx/edx-enterprise/blob/master/enterprise/management/commands/seed_enterprise_devstack_data.py#L47). Usually this means running something like `./manage.py lms seed_enterprise_devstack_data`, but refer to the link for latest info
- - Ensure the Enterprise Integration flag is enabled in devstack [see this link](https://github.com/edx/edx-platform/blob/master/lms/envs/devstack.py#L326)
- - Restart any services such as `make lms-restart` for applying changes
- - Pick a user such as enterprise_learner_1 to use as learner account (password is the same as the test edx account documented on devstack)
- - Go to http://localhost:8734/test-enterprise/ to browse the application for the `test-enterprise` customer
+
+## Setup enterprise stack
+
+An enterprise portal will need a couple of roles: The Enterprise customer, and at least one learner account.
+We also need the edx-enterprise Django module to be running inside of the LMS (which is started by devstack). Refer to [Devstack Doc](https://github.com/edx/devstack) for details. This is what serves endpoints used by the application.
```

Comment:
As mentioned by @taliaedX and my comments below, a published version of edx-enterprise is already installed into the LMS.